edx/edx-enterprise
# Added new field ``reply_to`` in enterprise customer which will be used in emails as from email address.

**Description:**
Added new field ``reply_to`` in enterprise customer which will be used in emails as from email address.

**Testing Instructions:**
This PR can be tested by adding a reply_to email address at the following URL
http://localhost:18000/admin/enterprise/enterprisecustomer/{uuid}

Then please check the following API
http://localhost:18000/enterprise/api/v1/enterprise-customer/{uuid}/

you will see the newly added reply_to email address in the API.

**Merge checklist:**
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.

**Post merge:**
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.
